code/game/machinery/requests_console.dm

```diff
@@ -407,18 +496,32 @@ var/list/obj/machinery/requests_console/requests_consoles = list()
 			screen = 0
 
 	//Handle silencing the console
-	switch( href_list["setSilent"] )
+	switch( href_list["toggleSilent"] )
 		if(null)	//skip
```

Comment:
this switch statement is not your fault, but the implementation is terrifying, this entire tree is set up wrong - and as a result you've duplicated the bad practice

look at basically any other topic() in the codebase for an example of how to do this correctly, or follow up with me if you are still confused